### DISCUSSION TITLE
[Pre-Release] 9.3.8c

### DISCUSSION THREAD

**Discussion by alienatedsec** [alienatedsec/yi-hack-v5#257] [Announcements]
## What's Changed
* Audio Testing by @alienatedsec in https://github.com/alienatedsec/yi-hack-v5/pull/213
* Pulling Audio and PWD Support by @alienatedsec in https://github.com/alienatedsec/yi-hack-v5/pull/218
* Audio Integration by @alienatedsec in https://github.com/alienatedsec/yi-hack-v5/pull/209
* Multiple Fixes by @alienatedsec in https://github.com/alienatedsec/yi-hack-v5/pull/242
* LAN IP address by @alienatedsec in https://github.com/alienatedsec/yi-hack-v5/pull/243

@roleoroleo has done the majority of audio and Home Assistant integration improvements

Instructions are the same as in this discussion https://github.com/alienatedsec/yi-hack-v5/discussions/248#discussion-5090628

Some changes between the last official 0.3.7 and 9.3.8c are below.
**Full Changelog**: https://github.com/alienatedsec/yi-hack-v5/compare/0.3.7...9.3.8c

<hr /><em>This discussion was created from the release <a href='https://github.com/alienatedsec/yi-hack-v5/releases/tag/9.3.8c'>[Pre-Release] 9.3.8c</a>.</em>

**Comment by Jakdaw** [comment]
I tried this on my outdoor camera this morning, and it didn't reappear on the network after the flashing. Not sure what went wrong, will report back here once I've got a ladder out to investigate what's happened to the camera.

**Reply by Jakdaw** [reply]
I've added details here: https://github.com/alienatedsec/yi-hack-v5/discussions/248
